Modules/Devices/test/Devices.Application.Tests/Tests/Devices/Commands/DeleteDevice/HandlerTests.cs

```diff
@@ -99,7 +99,7 @@ private static Device CreateOnboardedDevice(Identity identity = null)
         return onboardedDevice;
     }
 
-    private static Handler CreateHandler(IIdentitiesRepository mockIdentitiesRepository, IUserContext? fakeUserContext = null)
+    private static Handler CreateHandler(IIdentitiesRepository mockIdentitiesRepository, IUserContext fakeUserContext = null)
```

Comment:
Why did you remove the ? here? The default value already shows that it CAN be null